process_files.go

```diff
@@ -107,6 +107,18 @@ func (v *Venom) readFiles(filesPath []string) (err error) {
 
 		ts.Name += " [" + f + "]"
 
+		if ts.Version == "2" {
```

Comment:
if not startwith 1 will be better and will avoid to modify this line for the next version